fhir-server/src/main/java/com/ibm/fhir/server/util/FHIRRestHelper.java

```diff
@@ -100,7 +100,9 @@
 
     private static final String EXTENSION_URL = "http://ibm.com/fhir/extension";
     private static final String LOCAL_REF_PREFIX = "urn:";
-
+    private static final boolean SKIP_VALIDATION = true; // constant for when we want to avoid repeating validation
+    private static final boolean NO_SKIP_VALIDATION = false; // do not skip validation
```

Comment:
Since this is used in two spots, perhaps it should be reconciled into one constant

Reply:
cleaned up